trigger.info.yml

```diff
@@ -3,3 +3,6 @@ description: 'Enables actions to be fired on certain system events, such as when
 package: Trigger
 core: 8.x
 type: module
+
+dependencies:
+  - drupal:action
```

Comment:
Module does not depend on action because action manager is a part of system, please remove that

Actially action module provides just ui & configurable actions extension